File: src/main/java/com/p2p/lending/service/BiaoService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Biao;
import com.p2p.lending.entity.Borrowmoney;

import com.github.pagehelper.PageInfo;

public interface BiaoService {

	public List<Biao> findList(Map<String, Object> map);

	int delete(Integer id);

	int create(Biao biao);

	Biao get(Integer id);

	int update(Biao biao);
}

File: src/main/java/com/p2p/lending/service/BiaoService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Biao
import org.springframework.stereotype.Service

@Service
interface BiaoService {

    fun findList(map: Map<String, Any>): List<Biao>

    fun delete(id: Int?): Int

    fun create(biao: Biao): Int

    operator fun get(id: Int?): Biao

    fun update(biao: Biao): Int
}
